client/landing/stepper/declarative-flow/internals/steps-repository/pattern-assembler/style.scss

```diff
@@ -304,6 +304,10 @@ $font-family: "SF Pro Text", $sans;
 		overflow-y: auto;
 	}
 
+	.screen-container__body--no-padding {
+		padding: 0;
```

Comment:
This makes the body of the screen container overflow. Maybe we have to add something like `flex-grow: 1` to the `screen-container__body`

![image](https://user-images.githubusercontent.com/13596067/226555127-34d04ec4-6494-4cee-95a7-18e5ba00e198.png)

Reply:
No idea why the scrollbar appears. Fixed in https://github.com/Automattic/wp-calypso/pull/74701